integr8ly/integreatly-operator
# Add new CRD for customer configuration

# Description
Add new CRD for customer configuration. This will allow the customer to mange the timing of their upgrades of RHMI, amongst other things

## Type of change
- New feature

## How to Verify
Ensure the RHMI operator is running

Wait a short while for the RHMIConfig reconciler to run. During the bootstrap phase a default customer config CR should be created.

Retrieve the RHMIConfig resource: 
`oc get RHMIConfig rhmi-configuration -o yaml`

Ensure the CR exists and has set the necessary default values:
```
spec.upgrade.alwaysImmediately = false
spec.upgrade.duringNextMaintenance = false
```
